test/Test/Data/Mutable/Set.hs

```diff
@@ -29,13 +29,10 @@ mutSetTests = testGroup "Mutable set tests" group
 group :: [TestTree]
 group =
   -- Tests of the form [accessor (mutator)]
-  [ testProperty "∀ x. member (sing x) x = True" memberSing1
-  , testProperty "∀ x,y/=x. member (sing x) y = False" memberSing2
-  , testProperty "∀ x. member (insert s x) x = True" memberInsert1
+  [ testProperty "∀ x. member (insert s x) x = True" memberInsert1
   , testProperty "∀ x,y/=x. member (insert s x) y = member s y" memberInsert2
   , testProperty "∀ x. member (delete s x) x = False" memberDelete1
   , testProperty "∀ x,y/=x. member (delete s x) y = member s y" memberDelete2
-  , testProperty "∀ x. size (sing x) = 1" sizeSing
```

Comment:
This could easily be replaced by `size empty = 0, couldn't it?

Reply:
Yes!